services/vehicle/src/main/java/com/wedriveu/services/vehicle/entity/VehicleStoreImpl.java

```diff
@@ -12,24 +13,25 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Date;
+import java.util.List;
 
 public class VehicleStoreImpl implements VehicleStore {
 
     @Override
-    public void mapEntityToJson() {
-        Vehicle vehicle = createDummyObject("MACCHINA1",
+    public void createVehiclesFile() {
```

Comment:
This method is the one who create the vehicles file at the start of our service. Unless we have the dynamic configuration file that adds vehicles at will, we must create the file first, with this.